extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {isExperimentOn} from '../../../src/experiments';
+import {dev} from '../../../src/log';
+import {Layout} from '../../../src/layout';
+import {CSS} from '../../../build/amp-sticky-ad-0.1.css';
+
+/** @const */
+const EXPERIMENT = 'amp-sticky-ad';
+/** @const */
+const TAG = 'amp-sticky-ad';
```

Comment:
lets just remove experiment and use TAG as well for the experiment name. these two used to be different but this was actually your first task to rename camelCase to dash-case

https://github.com/ampproject/amphtml/pull/2837/files